mmhuman3d/data/data_converters/coco.py

```diff
@@ -18,7 +24,11 @@ class CocoConverter(BaseConverter):
     <https://arxiv.org/abs/1405.0312>`__ .
     """
 
-    def convert(self, dataset_path: str, out_path: str) -> dict:
+    def convert(self,
+                dataset_path: str,
+                out_path: str,
+                multi_human_data: bool = False,
```

Comment:
Bool type var `multi_human_data` has a lower name of class `MultiHumanData`, it looks like an instance of  class `MultiHumanData`. I suggest something like `use_multi_human_data` or ` enable_multi_human_data`.

Reply:
done